src/components/AddPlayerButton.js

```diff
@@ -0,0 +1,25 @@
+import React, { Component } from "react";
+import { ContentCardAction } from "./ContentCard";
+
+class AddPlayerButton extends Component {
+    constructor(props) {
+        super(props);
+
+        this.handleSubmit = this.handleSubmit.bind(this);
+    }
+
+    handleSubmit(event) {
+        this.props.addingPlayer(true);
+        event.preventDefault();
+    }
+
+    render() {
+        return (
+            <form onSubmit={this.handleSubmit}>
+                <ContentCardAction>Add Player</ContentCardAction>
```

Comment:
The ContentCardAction is specific to the ContentCard so it may get styles that we don't want on every button so I wouldn't use it outside of the ContentCard. Writing a component type that has specific inner components is a good pattern so that you don't have to control the layout but have control over the overall style no matter how the dev writing it decides to make it (in the ContentCard case you could have a content card with the actions at the top or in the middle of two texts and the styles should hold